css/css-sizing/aspect-ratio/grid-aspect-ratio-018.html

```diff
@@ -17,6 +17,9 @@
 <p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
 
 <div id="reference-overlapped-red"></div>
-<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 100px;">
-  <div style="height: 100px; aspect-ratio: 1/2; justify-self: stretch; background: green;"></div>
+<div style="display: grid; grid-template-columns: 100px; grid-template-rows: 50px;">
```

Comment:
Why we need to change the grid-template-rows value to 50px ?

Reply:
To keep displaying 100x100 green rectangles like before.